mailpoet/RoboFile.php

```diff
@@ -311,6 +311,10 @@ public function testPerformance($path = null, $opts = ['url' => null, 'pw' => nu
       ->option('env', 'PW=' . $opts['pw'])
       ->option('env', 'HEADLESS=' . ($opts['head'] ? 'false' : 'true'))
       ->option('env', 'SCENARIO=' . $opts['scenario'])
+      ->option('env', 'K6_CLOUD_TOKEN=' . getenv('K6_CLOUD_TOKEN'))
+      ->option('env', 'K6_CLOUD_ID=' . getenv('K6_CLOUD_ID'))
+      ->option('env', 'K6_PROJECT_NAME=' . $opts['name'])
+      ->option('out', 'cloud')
```

Comment:
What will happen when we run this locally? In that case, will it also try to log to the cloud? Or if I have no credentials, will that just skip it or something?

Reply:
There will be error in the output that you need to have authorization token. The token should be placed in the `.env`, currently present in the secret store along with all other vars.

Reply:
And, if you want to run locally you should probably want to have it under different project name. So, you would add --name=localjanjakes for example.. otherwise, it will be ran under pullrequests. I will add this info in ReadMe file in a separate PR to update ReadMe with all other required info.

Reply:
@veljkho This is not good. The defaults should not be difficult to use, and I think there's no reason to log local runs to the cloud.

Let's maybe try something like "if `K6_CLOUD_ID` env var doesn't exist, don't log anything`, or similar setup, where by default, it would not use the cloud at all.

Reply:
You were right, thanks for suggestion!